dnf.spec

```diff
@@ -96,6 +96,31 @@ Summary:        As a Yum CLI compatibility layer, supplies /usr/bin/yum redirect
 %description yum
 As a Yum CLI compatibility layer, supplies /usr/bin/yum redirecting to DNF.
 
+%package python2-yum
```

Comment:
I am not sure how to name it. pythonx.dnf-yum, but may be better idea will come.